lom/src/lib/lib_test/config/tx2_test.go

```diff
@@ -186,14 +186,13 @@ func TestGoroutineTracker(t *testing.T) {
 
         // Check that the goroutine is no longer running after waiting for it
         wg.Wait()
-        running, _ = mygoroutinetracker.IsRunning("test_goroutine5")
         time.Sleep(1000 * time.Millisecond)
```

Comment:
Be conservative about sleep. Better have other ways as all these sleeps will cumulate into long running UT, where we  could find hard to distinguish hang vs slow-progressing.

Reply:
Yes. I reduced it to 100ms. Here its testing for negative scenario until goroutine expires.